components/form/Textarea.js

```diff
@@ -0,0 +1,29 @@
+export default function Textarea({
+    name,
+    placeholder,
+    label,
+    disabled = false,
+    ...restProps
+  }) {
+    return (
+      <>
+        {label && (
+          <label htmlFor={name} className="mt-4 mb-3">
+            {label}
+          </label>
+        )}
+        <textarea
+          rows={4}
+          placeholder={placeholder}
+          className={`border-2  transition-all duration-250 ease-linear rounded  pr-6 py-2 mb-2 block w-full  ${
+            disabled
+              ? "bg-primary-low-medium dark:bg-primary-medium-low hover:border-primary-medium-low focus:ring-0 focus:border-primary-medium focus:outline-0 cursor-not-allowed"
+              : "dark:bg-primary-high hover:border-tertiary-medium focus:ring-0 focus:border-tertiary-medium focus:outline-0"
+          }"}`}
+          id={name}
+          name={name}
+          {...restProps}
+        />
+      </>
+    );
+  }
```

Comment:
Blank line ending required...

```suggestion
}

```